src/cap-description.adoc

```diff
@@ -81,6 +81,19 @@ The byte-address of a memory location is encoded as MXLEN integer value.
 ^| 64      ^| {cap_rv64_addr_width}
 |==============================================================================
 
+[#section_cap_level,reftext="capability level"]
+==== Capability Level (CL)
+Capabilities can be assigned a level, which in conjunction with the <<sl_perm>> and <<ll_perm>> allows enforcing invariants on capability propagation.
+For example, this can be used to ensure that a callee cannot save a copy of the passed in argument or to avoid sharing of compartment-local data between compartments.
+
+The Capability Level has two values: when set to 1 the capability is _global_ (in general allowing it to be stored using any authorizing capability), and when set to 0 the capability is _local_, and can only be stored by authorizing capabilities with the <<sl_perm>> set.
+Furthermore, the <<ll_perm>> can be used to restrict loading of _global_ capabilities -- causing the loaded value to be _local_ instead.
```

Comment:
> causing the loaded value to be _local_ instead

Is this similar to the LoadMutable permission in that the hardware automatically changes the CL of the capability read from memory to be 0 (i.e. local)?

Reply:
yes exactly, it behaves like LoadMutable.